### DISCUSSION TITLE
update how to install requirements, and systems tested on

### DISCUSSION THREAD

**Discussion by kanarelo** [alexmyczko/ruptime#16] [General]
We might need to update the readme file so that it can show how to install the items on specific platforms

- Ubuntu
- Debian
- Mac OSX

Also update the list of systems supported + tested on.

**Comment by alexmyczko** [comment]
true, for debian/ubuntu, i'm waiting for https://ftp-master.debian.org/new.html 

and for macOS i will have a brew once i'm ready: https://github.com/alexmyczko/homebrew-mac
(mcrypt is not installable anymore and will be replaced by openssl)

**Comment by alexmyczko** [comment]
ok for macOS you can do the following
```
wget https://raw.githubusercontent.com/alexmyczko/homebrew-mac/main/ruptime.rb
brew install ./ruptime.rb
```

configure the client
```
$ defaults read ch.aiei.ruptime
{
    HOSTNAMECMD = "echo papaya.local";
    KEY = PUBLICINTERNETRUPTIMEKEYNOTSUITABLEFORPRIVATEDATAYOUHAVEBEENWARNED;
    PORT = 51300;
    SERVER = "bananas.debian.net";
    SPOOL = "/var/spool/ruptime";
}
```
so run `defaults write ch.aiei.ruptime KEY VALUE`

update ruptime every minute, https://launchd.info
```
FILLME
```

rhw: missing date

rdisk: `diskutil list |grep physical |awk '{print $1}'`; then `diskutil info /dev/disk0 |grep "Disk Size:"`

rnet: fix wifi and ethernet output

**Comment by alexmyczko** [comment]
for freebsd port, see https://docs.freebsd.org/en/books/porters-handbook/quick-porting/

```
alles in /etc hardcoded? -> freebsd forbidden -> 3rd party -> $PREFIX/etc (default: /usr/local/etc)
sed patterns nicht kompatibel mit bsd-sed (gsed ist in den ports, binary heisst aber gsed)
sysctl tunabeles die es auf FreeBSD nicht gibt:
# sysctl -n hw.memsize
sysctl: unknown oid 'hw.memsize'
```

also #!/bin/bash, fix like with fnt port

**Comment by alexmyczko** [comment]
I have also added instructions for Alpinelinux.
